package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "axios": "^0.19.0",
     "core-js": "^2.6.5",
+    "cron": "^1.7.2",
```

Comment:
As I said before, please remove cron dependency,
We will let the host decide when to run.